internal/sanitizers.h

```diff
@@ -16,17 +16,11 @@
 #endif
 
 #ifdef HAVE_SANITIZER_ASAN_INTERFACE_H
-# if __has_feature(address_sanitizer)
-#  define RUBY_ASAN_ENABLED
-#  include <sanitizer/asan_interface.h>
-# endif
+# include <sanitizer/asan_interface.h>
 #endif
 
 #ifdef HAVE_SANITIZER_MSAN_INTERFACE_H
-# if __has_feature(memory_sanitizer)
-#  define RUBY_MSAN_ENABLED
```

Comment:
These macros, `RUBY_ASAN_ENABLED` and `RUBY_MSAN_ENABLED`, are used in many places.
They are all unnecessary?

Reply:
These are also defined in autoconf by this patch, which is why I deleted them from here.